aws/oidc/oidc.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+  plan_name  = "gh_plan_role"
+  admin_name = "gh_admin_role"
+}
+
+
+module "oidc" {
```

Comment:
need to add `oidc_exists = false` in-order to create the identity provider since the default assume the project resides in the new control tower setup that auto creates this.